packages/mobile/src/pincode/authentication.ts

```diff
@@ -276,6 +277,12 @@ export function* getPasswordSaga(account: string, withVerification?: boolean, st
 
 type PinCallback = (pin: string) => void
 
+async function addBiometryVerificationDelay() {
+  // insert artificial delay so that the native biometry verification
+  // animation can run the full course
+  return new Promise((resolve) => setTimeout(resolve, BIOMETRY_VERIFICATION_DELAY))
+}
+
```

Comment:
I noticed we already have 2 helpers that can do this:

https://github.com/valora-inc/wallet/blob/aaa9de2af0e32814f214f5b846839c2d9ed7826c/packages/mobile/src/geth/saga.ts#L2

https://github.com/valora-inc/wallet/blob/57cd5a341cbffc62903380dffbbb09850f6ec828/packages/mobile/src/pincode/authentication.ts#L12

We should pick one and remove the other (removal in a different PR) 😅

Reply:
noooo i didn't even see there was a sleep function right there 🤦🏻‍♀️ i went with the sleep from the @celo project because it's used in more places (and we also use a retry function from there) so feels like it's easier to remove `sleep-promise`